build.sh

```diff
@@ -76,10 +83,19 @@ VERSION=$(head -n 1 VERSION)
 case $VERSION in
   *SNAPSHOT)
     echo "Snapshot version (${VERSION}) specified in $(find . -name VERSION): building nightly plugin registry."
-    docker build -t "${IMAGE}" -f ${DOCKERFILE} --target ${TARGET} .
+    docker build \
+        -t "${IMAGE}" \
+        -f ${DOCKERFILE} \
+        --build-arg "USE_DIGESTS=${USE_DIGESTS}" \
```

Comment:
does not work for Brew builds. Need another approach, like using an ENV var or multistage.

Reply:
I don't know that it's possible to pass an env var into the docker context, `build-arg` seems to be the only option for setting build-time variables.

Reply:
multistage would work here too, but be clunky for upstream.